tableau-specialist/1.0/modules/ROOT/pages/tableau-specialist-connector-reference.adoc

```diff
@@ -0,0 +1,1255 @@
+| Project ID a| String |  ID of the project to delete. |  | x
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. The default is to use repeatable streams. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| Binary
+|===
+
+=== For Configurations
+
+* <<TableauSpecialistConfig>>
+
+=== Throws
+
+* TABLEAU-SPECIALIST:CONNECTIVITY
+* TABLEAU-SPECIALIST:DELETION_FORBIDDEN
+* TABLEAU-SPECIALIST:FORBIDDEN
+* TABLEAU-SPECIALIST:INSUFFICIENT_SITE_STORAGE_REMAINING
+* TABLEAU-SPECIALIST:PROJECT_NOT_FOUND
+* TABLEAU-SPECIALIST:RETRY_EXHAUSTED
+* TABLEAU-SPECIALIST:TIMEOUT
+
+
+[[InitiateFileUpload]]
+== Initiate file upload
+`<tableau-specialist:initiate-file-upload>`
+
+
+Initiates the upload process for a file.
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. The default is to use repeatable streams. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| Binary
+|===
+
+=== For Configurations
+
+* <<TableauSpecialistConfig>>
+
+=== Throws
+
+* TABLEAU-SPECIALIST:ACCESS_TO_FAVORITES_FORBIDDEN
+* TABLEAU-SPECIALIST:CONNECTIVITY
+* TABLEAU-SPECIALIST:FORBIDDEN
+* TABLEAU-SPECIALIST:INSUFFICIENT_SITE_STORAGE_REMAINING
+* TABLEAU-SPECIALIST:RETRY_EXHAUSTED
+* TABLEAU-SPECIALIST:TIMEOUT
+
+
+[[PublishDatasource]]
+== Publish datasource
+`<tableau-specialist:publish-datasource>`
+
+
+Publishes a data source onto the site, or appends data onto an existing data source. To make other changes to a published data source, call *Update Data Source* or *Update Data Source Connection*.
```

Comment:
onto > to
I am just curious, where do users specify the site?

Reply:
Demo says you specify /src/main/resources/project-properties.properties